.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Clone python template into devContainer to act as a testing playground for CLI development
+echo "#######################################################"
+echo "###        Clone vehicle-app-python-template        ###"
+echo "###         to act as a testing playground          ###"
+echo "###               for CLI execution                 ###"
+echo "#######################################################"
+
+rm -rf vehicle-app-repo/
+git clone https://github.com/eclipse-velocitas/vehicle-app-python-template.git vehicle-app-repo
+
+# add repo to git safe.directory
+REPO=$(pwd)
+git config --global --add safe.directory $REPO
+
+echo "#######################################################"
+echo "###  Install prerequisites for development of CLI   ###"
+echo "###    Node, build-essentials, npm dependencies     ###"
+echo "#######################################################"
+
+sh ./install_node.sh
```

Comment:
Maybe we need to have tasks.json to build/re-build velocitas-cli  each time then we deliver changes?

Reply:
How do you mean this exactly?
For the development in the devcontainer you actually dont really need to build the app/binary/cli since we can use the /bin/dev binary which dynamically compiles/transpiles the typescript codebase (thats also why the execution of the velocitas commands are taking a bit longer than when running with the binary directly)

Reply:
Let's make demonstration session :)